src/main/java/dao/AddToStorage.java

```diff
@@ -0,0 +1,5 @@
+package dao;
+
+public interface AddToStorage {
+    void writeToStorage(String fruit, Integer number);
+}
```

Comment:
I think you can add all methods which are used to work with database in single interface

Reply:
if the storage change, we can just creat new implementetions of this interfaces or use some old ones.